experimental/InvariantReductiveGroups/InvariantRings_ReducetiveGroups.jl

```diff
@@ -0,0 +1,478 @@
+import Oscar.gens, AbstractAlgebra.direct_sum
+
+mutable struct InvariantRing
+    field::QQField
+    poly_ring::MPolyDecRing #grade
+    group::Tuple{Symbol,Int64}
+    group_equations::MPolyRingElem
+    group_rep:: T where T <: AbstractAlgebra.Generic.MatSpaceElem
```

Comment:
```suggestion
    group_rep::AbstractAlgebra.Generic.MatSpaceElem
```
The `T` does not do anything useful.